drivers/network/dd/CMakeLists.txt

```diff
@@ -3,3 +3,4 @@ add_subdirectory(e1000)
 add_subdirectory(ne2000)
 add_subdirectory(pcnet)
 add_subdirectory(rtl8139)
+add_subdirectory(netkvm)
```

Comment:
Please add a new line at the end of file.

Reply:
Thanks, I fixed.